kong/reports.lua

```diff
@@ -114,7 +114,7 @@ end
 
 local function send_report(signal_type, t, host, port)
   if not _enabled then
-    return
+    return _enabled
```

Comment:
Could we keep the original code `return`? it means `return nil`, which should work as `return _enabled`

Reply:
I did it on purpose, I think we can distinguish the different returning behaviors between not_enabled and successfully. I added a test case, if it was kept original without being changed, the test case would still work by a chance even if the logic we added doesn't work as expected.

Reply:
could you add a comment about it? or else others will be confused tool

Reply:
or we return `false`?

Reply:
It is better to return false